omniscidb/QueryEngine/Compiler/HelperFunctions.cpp

```diff
@@ -101,14 +103,62 @@ void eliminate_dead_self_recursive_funcs(
 }
 #endif
 
+std::string legalizePassName(const llvm::StringRef pass_name) {
+  std::string legal_name = pass_name.str();
+  constexpr std::string_view illegal_word = "w/";
+  const size_t illegal_word_pos = legal_name.find(illegal_word);
+  if (illegal_word_pos != std::string::npos) {
+    std::string replacement =
+        "with" + legal_name.substr(illegal_word_pos + illegal_word.size());
+    legal_name.replace(illegal_word_pos, replacement.size(), replacement);
+  }
```

Comment:
Could you please add a comment describing the problem here? Mb add a couple of tests as well.